projects/storefrontlib/src/cms-components/product/product-variants/guards/product-variant.guard.spec.ts

```diff
@@ -50,8 +50,8 @@ class MockProductService {
 
 describe('ProductVariantGuard', () => {
   let guard: ProductVariantGuard;
-  let productService: ProductService;
-  let routingService: RoutingService;
+  let productService: MockProductService;
+  let routingService: MockRoutingService;
```

Comment:
The type of these vars should not be the Mocks, but the non mocks there was previously.